deploy/awstools/awstools.py

```diff
@@ -202,14 +204,16 @@ def awsinit() -> None:
         rootLogger.info("Running aws configure. You must specify your AWS account info here to use the FireSim Manager.")
         # DO NOT wrap this local call with StreamLogger, we don't want creds to get
         # stored in the log
+        # XXX confirm that StreamLogger doesn't log the interactive input of the sub-process and
+        # remove the above comment
```

Comment:
```suggestion
```

@filipstamenkovic-sifive can you confirm that when you run `firesim managerinit --platform f1` and it runs `awsinit()` here that the credentials given to `aws configure` are not logged into `logs/*managerinit*.log`?

Reply:
I just ran the script locally without real AWS credentials to check if my user input will be stored in the log, here's the output:

```2022-08-08 06:34:55,704 [managerinit ] [INFO ]  Adding default overrides to config files
2022-08-08 06:34:55,726 [_new_conn   ] [DEBUG]  Starting new HTTP connection (1): 169.254.169.254:80
2022-08-08 06:34:56,736 [_new_conn   ] [DEBUG]  Starting new HTTP connection (2): 169.254.169.254:80
2022-08-08 06:34:57,740 [awsinit     ] [INFO ]  Running aws configure. You must specify your AWS account info here to use the FireSim Manager.
2022-08-08 06:34:57,740 [flush       ] [DEBUG]  [localhost] local: aws configure
2022-08-08 06:35:41,536 [firesim_inpu] [CRITI]  If you are a new user, supply your email address [abc@xyz.abc] for email notifications (leave blank if you do not want email notifications): 
2022-08-08 06:35:45,509 [firesim_inpu] [DEBUG]  User Provided input():''
2022-08-08 06:35:45,509 [awsinit     ] [INFO ]  You did not supply an email address. No notifications will be sent.
2022-08-08 06:35:45,509 [<module>    ] [INFO ]  The full log of this run is:
/Users/filips/firesim/deploy/logs/2022-08-08--11-34-55-managerinit-GOMFPPG16ZNBBR7Z.log```

Looks like credentials did not get stored.